### PR TITLE
Redid lobby page completely

### PR DESCRIPTION
**Pull request by Ashpan**
image

### CODE REVIEW

**Comment by shoesax** [comment]
Minor nit: missing unit tests

File: app/(protected)/session/[code]/lobby-old/page.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  params: Promise<{ code: string }>;
+}) => {
+  const code = (await params).code;
+  const supabase = await createClient();
+  const {
+    data: { user },
+    error,
+  } = await supabase.auth.getUser();
+  if (!user || error) {
+    return;
```
Comment (shoesax) [inline app/(protected)/session/[code]/lobby-old/page.tsx]: Log error reason

Comment (shoesax) [reply app/(protected)/session/[code]/lobby-old/page.tsx]: pls :(

Comment (shoesax) [reply app/(protected)/session/[code]/lobby-old/page.tsx]: understood

File: components/ui/command.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
+"use client";
+
+import * as React from "react";
+import { type DialogProps } from "@radix-ui/react-dialog";
```
Comment (aarth19) [inline components/ui/command.tsx:4]: unecessary import... unused

**Pull request merged**